02_configure_host.sh

```diff
@@ -184,7 +184,7 @@ if [ "$EXT_IF" ]; then
 fi
 
 # Switch NetworkManager to internal DNS
-if [ "$MANAGE_BR_BRIDGE" == "y" ] ; then
+if [ "$MANAGE_BR_BRIDGE" == "y" ] ||  [ "$NODES_PLATFORM" = "assisted" ]; then
```

Comment:
MANAGE_BR_BRIDGE=y is the default, so why do you need the platform conditional here?

Reply:
I am setting it to false by default, at least for now i don't need MANAGE_BR_BRIDGE but this part of code i am still using.
And if somebody will set it to false and use assisted deployment i want to be sure this code will run. Building on it.